google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/prepare/sql_objects/assembled_sql_objects_entry_factory_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from google.cloud import datacatalog
+
+import mock
+
+from google.datacatalog_connectors.commons_test import utils
+from google.datacatalog_connectors.rdbms.prepare.sql_objects import \
+        assembled_sql_objects_entry_factory
+
+_PREPARE_PACKAGE = 'google.datacatalog_connectors.rdbms.prepare'
```

Comment:
Looks like this constant is redeclared at line 40. If so, remove this one, please.